cmd/config/docs/api-conventions/functions-impl.md

```diff
@@ -85,7 +85,7 @@ End-of-message
 `Dockerfile` installs `kustomize fn` and copies the script into the container image.
 
 ```
-FROM golang:1.18-stretch
```

Comment:
[stretch](https://wiki.debian.org/DebianStretch) is rather old and has stopped receiving security updates. [bullseye ](https://wiki.debian.org/DebianBullseye)is the current stable distribution

Reply:
Thanks for the comments. Good to know as well (like the `^` thing)